server/db/models/Log.js

```diff
@@ -9,7 +10,7 @@ const LogSchema = new Schema({
   },
   topic: {
     type: String,
-    enum: ["Medical", "Behavioral", "Other"],
+    enum: consts.topicArray,
     required: true,
   },
   tags: {
```

Comment:
This is on me, I didn't add an enum property in the mongoose Log model for `tags`, but we would also like to create a consts array for `["auditory", "heartworm", "fleas/ticks"]`. 

Could you change this tags definition to:
```js
tags: {
    type: [String],
    enum: ["auditory", "heartworm", "fleas/ticks"]
  }
```